numpy/version.py

```diff
@@ -0,0 +1,11 @@
+from ._version import get_versions
+
+__ALL__ = ['version', 'full_version', 'git_revision', 'release']
+
+vinfo = get_versions()
+version = vinfo["version"]
+full_version = vinfo['version']
+git_revision = vinfo['full-revisionid']
```

Comment:
```suggestion
version: str = vinfo["version"]
full_version: str = vinfo['version']
git_revision: str = vinfo['full-revisionid']
```
Would it be possible to add a few annotations here? This should be enough to annotate the entire module.
In principle we could also reintroduce `version.pyi`, but I feel this is change would be quite harmless.

Reply:
Sure. How do I do that?

Reply:
Oh, NVM, I see. I wondered why the original was done that way :)

Reply:
Done. I'll wait before pushing in case more fixes are suggested.